web/src/components/SidebarLink.js

```diff
@@ -1,9 +1,36 @@
 import PropTypes from 'prop-types';
 import React, { Fragment } from 'react';
+import Icon from '@fortawesome/react-fontawesome';
+
+import { faChevronDown, faChevronUp } from './Icons';
```

Comment:
not necessary, but can import the `Icon` component directly from `./Icons` (as the default) and save a line